### DISCUSSION TITLE
ATi Radeon 7200 Recalctimings SVGA

### DISCUSSION THREAD

**Discussion by wprball** [PCBox/PCBox#35] [Ideas]
I'd like to add ATi Radeon 7200 (R100) to this r100_recalctimings SVGA, because of low color modes.

Here's picture of R100 in Windows XP for this.
![20220406-162903-684](https://user-images.githubusercontent.com/80141462/162023236-ab65c808-8fc3-4ad9-aeef-0f590470b49c.png)

Add to there r100_recalctimings SVGA, such as ma_latch, rowoffset, and crtc.
![recalpicture](https://user-images.githubusercontent.com/80141462/162023923-eb7df8ed-a7fc-42a4-922e-5dc84aef769b.png)

**Comment by bachimusprime** [comment]
Why do you insist on using things that obviously aren't ready yet? This stuff's hidden for a reason.

**Reply by Torinde** [reply]
probably you're aware, but in case not: QEMU has the [2D part of Radeon RV100](https://www.kraxel.org/blog/2019/09/display-devices-in-qemu/#ati-vga)
